components/UI/titles/mainTitle.tsx

```diff
@@ -0,0 +1,34 @@
+import React, { FunctionComponent } from "react";
+import styles from "../../../styles/components/titles.module.css";
+import Button from "../../button";
+
+type MainTitleProps = {
+  title: string;
+  highlighted: string;
+  subtitle: string;
+  buttonName: string;
+  buttonLink: string;
+};
+
+const MainTitle: FunctionComponent<MainTitleProps> = ({
+  title,
+  highlighted,
+  subtitle,
+  buttonName,
+  buttonLink,
+}) => {
+  return (
+    <div className={styles.container}>
```

Comment:
Can you add a random image for me to be able to add it easily when we will have it (because) replace when we'll have it (Only on laptop).

Here is an image that you can use (you don't have to import it we'll use the link).

https://www.starknet.id/visuals/affiliates/affiliationIllu.webp